types/signed_msg_type.go

```diff
@@ -11,3 +23,14 @@ func IsVoteTypeValid(t cmtproto.SignedMsgType) bool {
 		return false
 	}
 }
+
+// From peppermint, will be eventually removed
```

Comment:
Consider removing it now: this isn't used anywhere. The only interest we could have in keeping this would be if we were doing DB migration, but it's not needed at all if we hard-fork